src/qz/printer/action/WebApp.java

```diff
@@ -359,9 +359,10 @@ private static synchronized void load(WebAppModel model, IntPredicate action) {
                 webView.setMaxHeight(1);
             }
 
+            //FIXME: These cause blank pages on raster
```

Comment:
@bberenz I get a blank first page on raster prints when this is added.  Is this needed for non-raster support or can it be removed?

Reply:
The transforms clear is needed for the vector prints since it manipulates them, but it can also be localized to just that path.